res/css/views/auth/_AuthBody.scss

```diff
@@ -152,3 +152,9 @@ limitations under the License.
         background-color: $accent-color;
     }
 }
+
+@media only screen and (max-width : 480px) {
```

Comment:
Please remove the space before the colon here and elsewhere. That should resolve the linting errors.